app/mailers/custom_devise_mailer.rb

```diff
@@ -7,4 +8,8 @@ def confirmation_instructions(record, token, options = {})
     @confirmation_period = user_decorator.confirmation_period
     super
   end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
+  end
```

Comment:
@amoose @brendansudol @monfresh I did some further testing and did find that using an external url from the asset pipeline worked reliably.  Using externally hosted images seems to be the standard practice of all of the larger email campaign services as well, so my vote would be to use that method rather than attachments.  With that said, since some of our environments are still using http auth, I left it as an attachment for now so images display correctly during testing.  In a future PR, I can switch it back to use the asset pipeline.